Multisig/UI/Transactions/TransactionDetails/LoadingTransactionDetailsViewModel.swift

```diff
@@ -73,4 +73,31 @@ class LoadingTransactionDetailsViewModel: ObservableObject {
 
         }
     }
+
+    func sign() {
+        guard let transferTx = transactionDetails as? TransferTransactionViewModel,
+              let transaction = transferTx.transaction else {
+            preconditionFailure("We have a technical problem. You need to restart the app.")            
```

Comment:
This will crash the app hard. The message should be "Failed to sign: either transaction is not a transfer or the internal transaction does not exist."

Reply:
ok